examples/custom_checkers/e9994_unnecessary_indexing.py

```diff
@@ -93,3 +93,20 @@ def loop_variable_reassigned(items: List[int]) -> int:
         s += items[i]
 
     return s
+
+
+def list_comp(lst: list) -> list:
+    """Return all the items in lst in a new list."""
+    return [lst[i] for i in range(len(lst))]  # Error on this line
+
+
+def comp_binop(lst: list) -> list:
+    """Return a list of all the items in lst multiplied by 2"""
+    return [2*lst[i] for i in range(len(lst))]  # Error on this line
+
+
+def comp_var_unused(lst: list) -> list:
+    """Iteration variable i is unused in the code, no unnecessary indexing performed"""
```

Comment:
Use the term "index variable", not "iteration variable". This occurs here and in the checker code itself (and possible the documentation file, so check that too).